components/edit_post/edit_post.tsx

```diff
@@ -152,7 +148,14 @@ const EditPost = ({editingPost, actions, ...rest}: Props): JSX.Element | null =>
         return () => {
             document.removeEventListener('paste', handlePaste);
         };
-    }, []);
+    }, []); // TODO This selector needs its dependencies passed correctly (see MM-42324)
+
+    const getEmojiContainerRef = useCallback(() => textboxRef.current, []);
+    const getEmojiTargetRef = useCallback(() => emojiButtonRef.current, []);
```

Comment:
Those are no dependencies, since what you depend on is in textboxRef and emojiButtonRef, and those are stable. If the `current` value changes, the reference is already taking care of returning the correct one.